pom.xml

```diff
@@ -93,6 +93,12 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
```

Comment:
Is this dependency related to the swagger documentation? Otherwise please remove.

Reply:
Yes, that dependency is for the lombok, which creates all the getters and setters for the DTOs

Reply:
I removed the lombok from this pom.xml